kubetest/main.go

```diff
@@ -841,7 +841,7 @@ func prepareAws(o *options) error {
 	if err := activateServiceAccount(o.gcpServiceAccount); err != nil {
```

Comment:
How about just

 `return activateServiceAccount(o.gcpServiceAccount)`

without the following

 `return nill`

?